perun-wui-core/src/main/java/cz/metacentrum/perun/wui/widgets/PerunDataGrid.java

```diff
@@ -692,6 +692,15 @@ public void filterTable(String text, PerunFilterEvent<T> filter) {
 	}
 
 	/**
+	 * Remove or add column to set.
```

Comment:
This is not a proper javadoc. Method has parameter and it should be described in it. Also name of method says, that is somehow updates the passed column, which is not true. Use something like "switchFilterOnColumn()" which says what method really does. Also please use proper formatting used in a rest of the code. You can setup you IDE to does this automatically.

From:
```
if (columnTypeSet.contains(perunColumnType)){
	columnTypeSet.remove(perunColumnType);
}else columnTypeSet.add(perunColumnType);
```
To:
```
if (columnTypeSet.contains(perunColumnType)) {
	columnTypeSet.remove(perunColumnType);
} else {
	columnTypeSet.add(perunColumnType);
}
```